Airgap Support VM custom SSL Certs (#2846)
spectrocloud/librarium: docs/docs-content/enterprise-version/install-palette/install-on-vmware/airgap-install/checklist.md

- [ ] Review the list of [pack binaries](../../airgap/supplemental-packs.md) to download and upload to your OCI
      registry.

> - [ ] If you have custom SSL certificates you want to include, copy the custom SSL certificates, in base64 PEM format,
>       to the support VM. The custom certificates must be placed in the **/opt/spectro/ssl** folder. Include the
>       following files:

Reviewer: ```suggestion
- [ ] If you want to include custom SSL certificates, copy them in the base64 PEM format
      to the **/opt/spectro/ssl** folder in your support VM. Include the
      following files:
```
